redisplus/json/commands.py

```diff
@@ -116,13 +119,8 @@ def jsonmget(self, path, *args):
         pieces.append(str_path(path))
         return self.execute_command("JSON.MGET", *pieces)
 
-    def jsonset(self, name, path, obj, nx=False, xx=False):
-        """
-        Set the JSON value at key ``name`` under the ``path`` to ``obj``.
-
-        ``nx`` if set to True, set ``value`` only if it does not exist
-        ``xx`` if set to True, set ``value`` only if it exists
-        """
+    def _executejsonset(self, name, path, obj, nx, xx):
```

Comment:
I don't think we need self._executejsonset any more, let's get that back into jsonset.